### PR TITLE
Add rolisteam v1.9.2

### PR DESCRIPTION
**Pull request by ChrisChinchilla**
After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.
- [x] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).

Additionally, if **adding a new cask**:

- [x] Named the cask according to the [token reference].
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests] for the same cask.
- [x] Checked the cask was not [already refused].
- [x] Checked the cask is submitted to [the correct repo].

[token reference]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/Homebrew/homebrew-cask/pulls
[already refused]: https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues
[the correct repo]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask

### CODE REVIEW

**Review by vladimyr — COMMENTED** [review]
Maybe it is a good idea to add character sheet editor cask too and point to it using caveats stanza 🤔

**Comment by vladimyr** [comment]
Also, you have an issue with SF URL format:
```
Cask 'rolisteam'.
audit for rolisteam: warning
 - SourceForge URL format incorrect. See https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/url.md#sourceforgeosdn-urls
```

File: Casks/rolisteam.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+cask 'rolisteam' do
+  version '1.9.2'
+  sha256 'eeb87c5cefc383845303edad44ea8b08ff808cb9e8ab80804a1172bbf1947a2d'
+
+  # sourceforge.net/projects/rolisteam/files/ was verified as official when first introduced to the cask
+
```
Comment (vladimyr) [inline Casks/rolisteam.rb:6]: Remove the blank line.

**Comment by ChrisChinchilla** [comment]
> Maybe it is a good idea to add character sheet editor cask too and point to it using caveats stanza 🤔

Doing that right now, the SF downloads for testing are sooo slow.

**Comment by ChrisChinchilla** [comment]
> Also, you have an issue with SF URL format:
> 
> ```
> Cask 'rolisteam'.
> audit for rolisteam: warning
>  - SourceForge URL format incorrect. See https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/url.md#sourceforgeosdn-urls
> ```

I followed the docs and ran all appropriate tests, if I find something incorrect, I'll submit a PR to the docs too.

**Comment by ChrisChinchilla** [comment]
Ahh, I think the appcast and URL differences confused me, sorry first time I've used cask with SF, I tend to avoid it.

**Comment by vladimyr** [comment]
I believe you but local audit command won't warn you about SF URL style. Take a quick look at the linked doc page.

**Comment by vladimyr** [comment]
> Ahh, I think the appcast and URL differences confused me, sorry first time I've used cask with SF, I tend to avoid it.

No worries, I feel your pain 😉

I believe you should now update the comment above download URL too.

**Review by vladimyr — APPROVED** [review]

**Pull request merged**